modules/space/rendering/renderablestars.h

```diff
@@ -1,26 +1,26 @@
 /*****************************************************************************************
- *                                                                                       *
```

Comment:
There are still some places where spaces have been unintentionally removed in these comments. Sorry for the nitpicking.